### PR TITLE
Group call added and update data center for Blueshift

### PR DESCRIPTION
**Pull request by ishachopra0401**
## Description of the change

Added updated implementation for group call

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

**Review by utsabc — APPROVED** [review]

**Review by akashrpo — APPROVED** [review]

**Review by sauravlal-rudderstack — APPROVED** [review]

File: destinations/marketing/blueshift.mdx [hunk 1]
```diff
@@ -189,6 +191,43 @@ Also, RudderStack automatically converts a space in the event name to an undersc
...
+| `context.idfa`/`context.device.advertisingId` | `device_idfa` | Optional|
+| `context.idfv`/`context.device.id` | `device_idfv` | Optional|
+| `context.device.manufacturer` | `device_manufacturer` | Optional|
+| `context.os.name` | `os_name` | Optional|
+| `context.network.carrier` | `network_carrier` | Optional|
+| `context.ip`, `request_ip` | `ip` | Optional|
+| `context.address.latitude`, `context.location.latitude` | `latitude` | Optional|
+| `context.address.longitude`, `context.location.longitude` | `longitude` | Optional|
+| `messageId` | `event_uuid` | Optional|
+
```
Comment (akashrpo) [inline destinations/marketing/blueshift.mdx:230]: We should add info for custom attributes as well. cc @sauravlal-rudderstack

Comment (sauravlal-rudderstack) [reply destinations/marketing/blueshift.mdx:230]: Users can add custom attributes, i.e.users can pass more information like cookie. It is directly mapped in Blueshift.

Comment (ishachopra0401) [reply destinations/marketing/blueshift.mdx:230]: @akashrpo @sauravlal-rudderstack  Does this require any changes in the doc?

Comment (sauravlal-rudderstack) [reply destinations/marketing/blueshift.mdx:230]: No, I think, we can add as a note, that users can add more information as custom attributes. CC @akashrpo

Comment (akashrpo) [reply destinations/marketing/blueshift.mdx:230]: @ishachopra0401 We need to add custom attribute info in the docs for group call

Comment (ishachopra0401) [reply destinations/marketing/blueshift.mdx:230]: done

File: destinations/marketing/blueshift.mdx [hunk 2]
```diff
@@ -52,7 +52,9 @@ To successfully configure Blueshift as a destination, you will need to configure
 For more information on obtaining the Event and Users API Key, refer to the <a href="#faq">FAQ</a> section below.
 </div>
 
-- **Blueshift EU data center**: Enable this option to send data to the EU servers. When enabled, the base URL for EU region converts to `https://api.eu.getblueshift.com`
+- **Data Center**: Select the data center where you want to send the data:
+  -  Standard (Default): The base URL is https://api.getblueshift.com
```
Comment (ameypv-rudder) [inline destinations/marketing/blueshift.mdx:56]:
```suggestion
  -  **Standard** (default): The base URL is https://api.getblueshift.com 
```

File: destinations/marketing/blueshift.mdx [hunk 3]
```diff
@@ -52,7 +52,9 @@ To successfully configure Blueshift as a destination, you will need to configure
 For more information on obtaining the Event and Users API Key, refer to the <a href="#faq">FAQ</a> section below.
 </div>
 
-- **Blueshift EU data center**: Enable this option to send data to the EU servers. When enabled, the base URL for EU region converts to `https://api.eu.getblueshift.com`
+- **Data Center**: Select the data center where you want to send the data:
+  -  Standard (Default): The base URL is https://api.getblueshift.com 
+  -  EU: The base URL is https://api.eu.getblueshift.com
```
Comment (ameypv-rudder) [inline destinations/marketing/blueshift.mdx:57]:
```suggestion
  -  **EU**: The base URL is https://api.eu.getblueshift.com
```

File: destinations/marketing/blueshift.mdx [hunk 4]
```diff
@@ -189,6 +191,48 @@ Also, RudderStack automatically converts a space in the event name to an undersc
 Blueshift supports receiving custom attributes about your site's customers. For more information on these attributes, refer to the <a href ="https://developer.blueshift.com/docs/customer-related-data#attributes">Blueshift Custom Attributes</a>.
 </div>
 
+## Group
+
+The `group` call lets you associate a user with a group. For each `group` call, the `event` API is triggered and an `identify` event is sent with the `groupId`. It then creates a group and adds that user to the group.
```
Comment (ameypv-rudder) [inline destinations/marketing/blueshift.mdx:196]:
```suggestion
The `group` call lets you associate a user with a group. For each `group` call, RudderStack triggers Blueshift's `event` API and sends an `identify` event along with the `groupId`. Blueshift then creates a group and adds the user to it.
```

Comment (ameypv-rudder) [reply destinations/marketing/blueshift.mdx:196]: Can we also add the link to the `event` API?

**Review by ameypv-rudder — APPROVED** [review]

**Pull request merged**